bigquery/datalab-migration/requirements.txt

```diff
@@ -5,4 +5,4 @@ google-cloud-bigquery[pandas,pyarrow]==2.34.4
 google-cloud-bigquery-storage===2.13.2
 datalab==1.2.0
 ipython==8.0.1
-pyarrow==6.0.1
+pyarrow==9.0.0
```

Comment:
Could this also be constrained to pyarrow<7 like the bqml one? If so, we should have a comment as well.

Reply:
Should all the `bigquery/` ones be constrained, or just this one?

Reply:
I'm not sure, that's just a guess. I'm not too familiar with bigquery, but I remember seeing that kind of constraint. It looks like the client library itself expects pyarrow<10 so maybe it should be fine?